File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/FieldsBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class FieldsBean {
    private String chnname;
    private String name;
    private String typeName;
    private String type;
    private String dataType;
    private String remark;
    private Boolean pk;
    private Boolean notNull;
    private Boolean autoIncrement;
    private Boolean relationNoShow;
    private String defaultValue;
    private String uiHint;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/FieldsBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class FieldsBean {
    var chnname: String? = null
    var name: String? = null
    var typeName: String? = null
    var type: String? = null
    var dataType: String? = null
    var rawDataType: String? = null
    var remark: String? = null
    var pk: Boolean? = null
    var notNull: Boolean? = null
    var autoIncrement: Boolean? = null
    var relationNoShow: Boolean? = null
    var defaultValue: String? = null
    var uiHint: String? = null
}
